site/en/docs/extensions/mv3/overview/index.md

```diff
@@ -114,10 +131,6 @@ or by pressing `Ctrl+Shift+F` on your keyboard.
 3.  Keep up to date by reading the [Chromium blog][12]
 4.  Subscribe to the [chromium-extensions group][13]
 
-## Featured videos {: #Featured-videos }
-
-[Technical videos][14]
-[Developer snapshots][15]
```

Comment:
+1 to dropping this.

Much broader topic, but later in the year (Q2 maybe?) we should consider what video content might be worth producing.